packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -62,9 +60,8 @@ export class ScGridTooltip {
     const color = this.breachedThreshold ? this.breachedThreshold.color : undefined;
     const displaysMoreThanTitle = thereIsSomeData && this.isEnabled;
     const icon = this.breachedThreshold ? this.breachedThreshold.icon : undefined;
-    const label = this.breachedThreshold ? this.breachedThreshold.label : undefined;
-    const unit = this.unit || '';
-    const value = this.value || this.propertyPoint?.y;
+    const label = this.breachedThreshold ? this.breachedThreshold.label?.text : undefined;
```

Comment:
Is the ternary necessary?

```suggestion
    const label = this.breachedThreshold?.label?.text;
```